src/Controller/JavascriptManagerController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Drupal\js_manager\Controller;
+
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\Core\Link;
+use Drupal\Core\Url;
+
+/**
+ * @file
+ * Contains \Drupal\js_manager\Controller\JavascriptManagerController.
+ */
+class JavascriptManagerController {
```

Comment:
We should see if we can use `Drupal\Core\Config\Entity\ConfigEntityListBuilder` or `Drupal\Core\Config\Entity\DraggableListBuilder`